### PR TITLE
Wifi 9227

### PR DESCRIPTION
**Pull request by dutta-rohan**

### CODE REVIEW

File: .github/workflows/interop_combined.yml [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+          CLOUDSHELL_PASSWORD: ${{ secrets.CLOUDSHELL_PASSWORD }}
+        run: |
+          python .quali/start_reservation.py --global-inputs '{"Phone ID(s)":"1,2,3,4,5,7,8","AP ID(s)":"1,2,3,4","Optional Existing SDK Namespace":"${{ needs.vars.outputs.existing_controller }}","Include a LANForge?":"Yes","securityToken":"eyJhbGciOiJIUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICI3NzkzZGM0Ni1jZmU4LTQ4ODMtYjhiOS02ZWFlZGU2OTc2MDkifQ.eyJpYXQiOjE2MzI4Mzc2NDEsImp0aSI6IjAwZGRiYWY5LWQwYjMtNDRjNS1hYjVlLTkyNzFlNzc5ZGUzNiIsImlzcyI6Imh0dHBzOi8vYXV0aDIucGVyZmVjdG9tb2JpbGUuY29tL2F1dGgvcmVhbG1zL3RpcC1wZXJmZWN0b21vYmlsZS1jb20iLCJhdWQiOiJodHRwczovL2F1dGgyLnBlcmZlY3RvbW9iaWxlLmNvbS9hdXRoL3JlYWxtcy90aXAtcGVyZmVjdG9tb2JpbGUtY29tIiwic3ViIjoiODNkNjUxMWQtNTBmZS00ZWM5LThkNzAtYTA0ZjBkNTdiZDUyIiwidHlwIjoiT2ZmbGluZSIsImF6cCI6Im9mZmxpbmUtdG9rZW4tZ2VuZXJhdG9yIiwibm9uY2UiOiI2ZjE1YzYxNy01YTU5LTQyOWEtODc2Yi1jOTQxMTQ1ZDFkZTIiLCJzZXNzaW9uX3N0YXRlIjoiYmRjZTFmYTMtMjlkYi00MmFmLWI5YWMtYjZjZmJkMDEyOTFhIiwic2NvcGUiOiJvcGVuaWQgcHJvZmlsZSBvZmZsaW5lX2FjY2VzcyBlbWFpbCJ9.5R85_1R38ZFXv_wIjjCIsj8NJm1p66dCsLJI5DBEmks"}' --reservation-duration 1440 "InterOp Lab - Specific Selection"
```
Comment (dunaev-opsfleet) [inline .github/workflows/interop_combined.yml:82]: Security token as a plaintext seems like a security issue

File: .github/workflows/interop_combined.yml [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+      - name: set reservation outputs
+        if: always()
+        id: quali_reservation
+        run: |
+          echo ::set-output name=identifier::"$(cat ./reservation_id.txt)"
+          echo ::set-output name=namespace::"$(cat ./reservation_id.txt | cut -d "-" -f 1)"
+#          echo "reservation_id=$(cat ./reservation_id.txt)" >> $GITHUB_ENV
```
Comment (dunaev-opsfleet) [inline .github/workflows/interop_combined.yml:89]: Comments cleaning is required

File: .github/workflows/quali.yml [hunk 3]
```diff
@@ -22,11 +22,11 @@ on:
         description: "revision of the Open Wifi Helm chart"
       ap_models:
         required: true
-        default: "Wallys_DR6018,hfcl_ion4xe,edgecore_eap101,edgecore_ecw5211,cig_wf188n,edgecore_eap102,liteon_wpx8324,hfcl_ion4,indio_um-305ac,hfcl_ion4xi,udaya_a5-id2,cig_wf196"
+        default: "wallys_dr40x9,hfcl_ion4xe,edgecore_eap101,tp-link_ec420-g1,edgecore_ecw5211,cig_wf188n,edgecore_eap102,cig_wf194c,hfcl_ion4,indio_um-305ac,hfcl_ion4xi,udaya_a5-id2,cig_wf196"
```
Comment (dunaev-opsfleet) [inline .github/workflows/quali.yml:25]: Are these changes fine? It seems that it doesn't belong here and overrides master branch changes

Comment (dutta-rohan) [reply .github/workflows/quali.yml:25]: Hi Dmitry, I shall redo this branch

File: .github/workflows/interop_combined.yml (hunk 1 above)
Comment (dutta-rohan) [reply .github/workflows/interop_combined.yml:82]: The same is currently being used in interop.yaml , would you suggest we add it to secrets?

**Pull request closed without merging**